src/components/Starred.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+import starredSlice from '../data/starredSlice'
+import Movie from './Movie'
+import '../styles/starred.scss'
+
+const Starred = ({viewTrailer}) => {
+
+    const state = useSelector((state) => state)
+    const { starred } = state
+    const { clearAllStarred } = starredSlice.actions
+    const dispatch = useDispatch()
+
+  return (
+    <div className="starred" data-testid="starred">
+      {starred.starredMovies.length > 0 && (<div data-testid="starred-movies" className="starred-movies">
+        <h6 className="header">Starred movies</h6>
+        <div className="row">
```

Comment:
I would recommend using the MoviesGrid component instead of manually mapping the movie elements (<Movie />). This improves readability and makes the code easier to reuse.